Models/InertDoubletModel/MatrixElements/idm.m

```diff
@@ -135,75 +135,53 @@
 
 
 (*left-handed top-quark*)
-ReptL=CreateParticle[{{1,1}},"F"];
+ReptL=CreateParticle[{{1,1}},"F", mq2, "TopL"];
 
 (*right-handed top-quark*)
-ReptR=CreateParticle[{{2,1}},"F"];
+ReptR=CreateParticle[{{2,1}},"F", mq2, "TopR"];
 
 (*light quarks*)
-RepLightQ = CreateParticle[{{1,2},3,6,7,8,11,12,13},"F"];
+RepLightQ = CreateParticle[{{1,2},3,6,7,8,11,12,13},"F", mq2, "LightQuark"];
 
 (*left-handed leptons*)
-RepLepL = CreateParticle[{4,9,14},"F"];
+RepLepL = CreateParticle[{4,9,14},"F", ml2, "LepL"];
 
 (*right-handed leptons -- these don't contribute*)
-RepLepR = CreateParticle[{5,10,15},"F"];
+RepLepR = CreateParticle[{5,10,15},"F", ml2, "LepR"];
 
 (*Vector bosons*)
-RepGluon=CreateParticle[{1},"V"];
+RepGluon=CreateParticle[{1},"V", mg2, "Gluon"];
 
 (*We are approximating the W and the Z as the same particle*)
-RepW=CreateParticle[{{2,1}},"V"];
+RepW=CreateParticle[{{2,1}},"V", mw2, "W"];
 
-(*Higgs*)
-RepHiggs = CreateParticle[{1},"S"];
+(*Higgs and Goldstones*)
+RepHiggs = CreateParticle[{1},"S", mh2,"Higgs"];
```

Comment:
Should this also renamed to HiggsDoublet?

Reply:
I could do it, but the comment already says it, no?